src/main/java/com/billyyccc/mssqlclient/impl/protocol/client/prelogin/OptionToken.java

```diff
@@ -0,0 +1,18 @@
+package com.billyyccc.mssqlclient.impl.protocol.client.prelogin;
+
+public abstract class OptionToken {
+  private final byte type;
+  int optionLength;
```

Comment:
can we make this final and passed in the constructor ?

Reply:
done in https://github.com/BillyYccc/incubator-vertx-mssql-client/pull/3/commits/afad9d21d54ead31d6e07eb0d4a1bc26087cdf60.